### PR TITLE
A0-3848: Update weekly nightly start FE tests to more recent aleph-node

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
We should be using more recent `aleph-node` images on our CI.

### CODE REVIEW

**Review by timorleph — APPROVED** [review]
Oh, definitely! Additionally, updating this should probably be part of the release procedure, no?

**Comment by Marcin-Radecki** [comment]
> Oh, definitely! Additionally, updating this should probably be part of the release procedure, no?

Could be yes, but I'd rather automate it, so e.g. make a step ni GHA that returns SHA/tag of current Testnet etc

File: .github/workflows/weekly-featurenets-short-session-full-sha.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
     secrets: inherit
     with:
       featurenet-name: 'ops-test-full-hash'
-      ref: '787c39186884aaf86ed48f46f575a8cd8cfd13e8'
+      ref: '87ebe7c4856b4f3365ddba55b55089713a96631a'
```
Comment (pmikolajczyk41) [inline .github/workflows/weekly-featurenets-short-session-full-sha.yml:27]: these sha are very opaque :{ how do you choose one?

Comment (Marcin-Radecki) [reply .github/workflows/weekly-featurenets-short-session-full-sha.yml:27]: This is just a sha of `r-13-3`. Agree is not intuitive what should be there, but we'll do a follow-up step that returns a tag/SHA from the current Testnet - in the next PR.

**Review by pmikolajczyk41 — APPROVED** [review]

**Pull request merged**